OWNERS_ALIASES

```diff
@@ -49,8 +49,8 @@ aliases:
   sig-contributor-experience-leads:
     - Phillels
     - parispittman
-    - grodrigues3
     - cblecker
+    - nikhita
```

Comment:
Doing a dump of OWNERS_ALIASES from k/community led to more changes (addition of wg-k8s-infra and wg-component-standard). Do you want me to 1) add them in this PR or 2) move this to another PR with those changes or 3) want to have any changes to `OWNERS_ALASES` in the next dump of changes to the file? :)